tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -253,6 +253,12 @@ public static void ClearProfilerEnvironmentVariables()
                 environmentVariables["DD_APPSEC_WAF_TIMEOUT"] = 10_000_000.ToString();
             }
 
+            // Setting the only propagation style to extract to Datadog to prevent changing current snapshots
+            if (!environmentVariables.ContainsKey("DD_TRACE_PROPAGATION_STYLE_EXTRACT"))
+            {
+                environmentVariables["DD_TRACE_PROPAGATION_STYLE_EXTRACT"] = "Datadog";
+            }
```

Comment:
I have a minor concern that this could cause us to miss tracestate/datadog interaction bugs in the integration tests 🤔 I wonder if the better option is to change the `VerifyHelper.ScrubTags` method to automatically exclude the `_dd.parent_id` values? 

https://github.com/DataDog/dd-trace-dotnet/blob/4d3d51d828c9ff345cf3cba268b0bc5e90714feb/tracer/test/Datadog.Trace.TestHelpers.SharedSource/VerifyHelper.cs#L198

Note that we have a couple of implementation we should update if we do that
- `VerifyHelper.ScrubTags`
- `VerifyHelper.ScrubCIVisibilityTags`

I can't remember if we have a couple of versions of VerifyHelper too - I think Debugger maybe has a version too?

Reply:
Updated at: https://github.com/DataDog/dd-trace-dotnet/pull/5449/commits/95eab1c50ef207be56af225d3ccf21c054bf926b